subsys/net/ip/Kconfig

```diff
@@ -139,6 +139,27 @@ config NET_TCP_ACK_TIMEOUT
 	various TCP states. The value is in milliseconds. Note that
 	having a very low value here could prevent connectivity.
 
+config NET_TCP_RETRY_COUNT
+	int "Maximum number of TCP fragment retransmissions"
```

Comment:
It would be better to talk about TCP segments here instead of fragments. The fragments term might get confused to net_buf fragments that is mentioned all over the place in the code.